tools/mob_suite/mob_recon.xml

```diff
@@ -1,25 +1,25 @@
-<tool id="mob_recon" name="MOB-Recon" version="1.4.9.1+galaxy0">
+<tool id="mob_recon" name="MOB-Recon" version="2.0.2+galaxy0">
```

Comment:
Perhaps update to version 2.0.5 as it brings the same features of the version 2.0.2 but solves some host range issues linked to non-ascii chars and mean transfer rate calculation?